addons/l10n_es_edi_facturae/models/l10n_es_edi_facturae_certificate.py

```diff
@@ -74,7 +74,12 @@ def _sign_xml(self, edi_data, signature_data):
         public_key_numbers = cert_public.public_key().public_numbers()
 
         rfc4514_attr = dict(element.rfc4514_string().split("=", 1) for element in cert_public.issuer.rdns)
-        issuer = f"CN={rfc4514_attr['CN']}, OU={rfc4514_attr['OU']}, O={rfc4514_attr['O']}, C={rfc4514_attr['C']}"
+
+        # The 'Organizational Unit' field is optional
+        if 'OU' in rfc4514_attr:
+            issuer = f"CN={rfc4514_attr['CN']}, OU={rfc4514_attr['OU']}, O={rfc4514_attr['O']}, C={rfc4514_attr['C']}"
+        else:
+            issuer = f"CN={rfc4514_attr['CN']}, O={rfc4514_attr['O']}, C={rfc4514_attr['C']}"
```

Comment:
Just a suggestion if multiple fields are optional
```suggestion
        issuer = ", ".join(
            f"{k}={rfc4514_attr[k]}" 
            for k in ['CN', 'OU', 'O', 'C']
            if k in rfc4514_attr
        )
```